carma_record/scripts/carma_record.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+while true
+    do
+        # Get the regex of the excluded topics
+        excluded_topics="$(rosparam get exclude_regex)"
+        if [ ! -z "${excluded_topics}" ]
+        then 
+            # Run the rosbag record command
+            rosbag record -o /opt/carma/logs/ --lz4 -a -x '/rosout(.*)|(.*)/received_messages|(.*)/sent_messages|(.*)/scan|$($excluded_topics)'
```

Comment:
Since there is a default option, I think it would make more sense to include this exclusions in that default set rather than hard coded.